deposit/src/main/java/edu/unc/lib/deposit/work/AbstractDepositJob.java

```diff
@@ -232,68 +208,23 @@ protected void verifyRunning() {
 	}
 
 	/**
-	 * Returns the PREMIS events file for the given PID
+	 * Creates new PremisLogger object from which instances can build and write Premis events to a file
 	 *
 	 * @param pid
-	 * @return
+	 * @return PremisLogger object
 	 */
-	protected File getEventsFile(PID pid) {
-		return createOrAppendToEventsFile(pid, null);
-	}
-
-	/**
-	 * Appends an event to the PREMIS log for the given pid. If the log does not exist, it is created
-	 *
-	 * @param pid
-	 * @param event
-	 */
-	protected void appendDepositEvent(PID pid, Element event) {
-		createOrAppendToEventsFile(pid, event);
-	}
-
-	/**
-	 * Appends an event to the PREMIS document for the given PID, creating the document if it does not already exist or
-	 * is corrupt.
-	 *
-	 * @param pid
-	 * @param event
-	 * @return the premis document file
-	 */
-	private File createOrAppendToEventsFile(PID pid, Element event) {
-		File file = new File(depositDirectory, DepositConstants.EVENTS_DIR + "/" + pid.getUUID() + ".xml");
+	public PremisLogger getPremisLogger(PID pid) {
+		File file = new File(depositDirectory, DepositConstants.EVENTS_DIR + "/" + pid.getUUID() + ".ttl");
 
 		try {
-			Document dom;
+			PremisLogger premisLogger;
 			if (!file.exists()) {
 				file.getParentFile().mkdirs();
-				dom = createNewEventsFile(pid, file);
-			} else {
-				// Not appending anything, so return before attempting to load existing file
-				if (event == null)
-					return file;
-
-				try {
-					dom = new SAXBuilder().build(file);
-				} catch (JDOMException e) {
-					log.warn("Failed to parse existing events file for {}, backing up corrupt log and creating a new log", e);
-					try {
-						Files.move(file.toPath(), Paths.get(file.getAbsolutePath() + ".backup." + System.currentTimeMillis()));
-					} catch (IOException e2) {
-						failJob(e2, "Failed to backup corrupt log file for object {}.", pid);
-					}
-					dom = createNewEventsFile(pid, file);
-				}
-			}
-
-			if (event != null) {
-				dom.getRootElement().addContent(event.detach());
-			}
-
-			try (FileOutputStream out = new FileOutputStream(file, false)) {
-				new XMLOutputter(Format.getPrettyFormat()).output(dom, out);
-			}
+				
+			} 
+			premisLogger = createNewEventsFile(pid, file);
```

Comment:
I think we can drop createNewEventsFile, all the method is doing is calling the constructor of PremisLogger which is doing the actual file creation